compiler/noirc_frontend/src/hir/comptime/mod.rs

```diff
@@ -1,3 +1,14 @@
+//! The comptime interpreter is used to execute Noir code which is marked to
+//! be executed at compilation time. The main uses for this are:
+//!     - Metaprogramming such as deriving trait implementations.
+//!     - Evaluation of globals.
```

Comment:
Should "Evaluation of comptime blocks" be included here? And also interpreting attribute functions?